exs/mundo_1/rust/033.rs

```diff
@@ -0,0 +1,58 @@
+/*
+Desafio 033
+
+Problema: Faça um programa que leia três números e mostre qual é o maior e qual é
+          o menor.
+*/
+
+fn main() {
+    let mut num1 = String::new();
+    let mut num2 = String::new();
+    let mut num3 = String::new();
+
+    println!("Informe o primeiro número");
+    std::io::stdin().read_line(&mut num1).unwrap();
+
+    println!("Informe o segundo número");
+    std::io::stdin().read_line(&mut num2).unwrap();
+
+    println!("Informe o terceiro número");
+    std::io::stdin().read_line(&mut num3).unwrap();
+
+    let num1 = num1
+        .trim()
+        .parse::<i32>()
+        .expect("Valor inserido não é um número válido.");
+
+    let num2 = num2
+        .trim()
+        .parse::<i32>()
+        .expect("Valor inserido não é um número válido.");
+
+    let num3 = num3
+        .trim()
+        .parse::<i32>()
+        .expect("Valor inserido não é um número válido.");
+
+    let mut menor = num1;
+    let mut maior = num2;
```

Comment:
Na lógica atual, o primeiro valor informado nunca será classificado como maior, visto que não há nenhuma verificação/atribuição para ele.

Para funcionar corretamente, pode-se assumir que o primeiro valor já é o maior:
```suggestion
    let mut maior = num1;
```